src/vaults/AsyncRequests.sol

```diff
@@ -428,8 +429,9 @@ contract AsyncRequests is BaseInvestmentManager, IAsyncRequests {
         PoolId poolId = vault_.poolId();
         ShareClassId scId = vault_.scId();
 
-        // TODO: Enable transient pricing for BS
-        // (D18 pricePoolPerAsset,) = poolManager.pricePoolPerAsset(poolId, scId, vaultDetails.assetId, true);
+        (D18 pricePoolPerAsset,) = poolManager.pricePoolPerAsset(poolId, scId, vaultDetails.assetId, true);
```

Comment:
NIT 

BS will use this Price.

Reply:
I don't understand this comment?

Reply:
If we have a default price in the BS, this will be it. So we technically dont need to override it here.